plugins/algoltek-usb/fu-algoltek-usb-device.c

```diff
@@ -463,19 +458,19 @@ fu_algoltek_usb_device_write_firmware(FuDevice *device,
 	fu_progress_add_step(progress, FWUPD_STATUS_DEVICE_WRITE, 95, NULL);
 
 	if (!algoltek_device_en(self, error)) {
-		g_prefix_error(error, "Err: ALGOLTEK_EN failure");
+		g_prefix_error(error, "ERR: ALGOLTEK_EN failure");
```

Comment:
this needs to be `"ALGOLTEK_EN failure: "` i.e. no ERR prefix, and a `: ` suffix (on all of these)

Reply:
Altered accordingly.